### ISSUE TITLE
Controls are not showing when calling .toggleControls(true)

### ISSUE THREAD

**Issue by philipgiuliani** [sampotts/plyr#883]
### Steps to reproduce
- Go to https://plyr.io/
- Play the video
- Type `player.toggleControls(true)` into the console

### Expected behaviour
Controls are shown when I call `player.toggleControls(true)`.

### Actual behaviour
Nothing happens when I call `player.toggleControls(true)`.

### Environment
- Browser: Chrome
- Version: 65
- Operating System: macOS
- Version: 10.12.6

**Comment by friday** [comment]
Assuming you mean "controls are __not__ showing". If this is the case I've submitted another issue and PR that relates to this.

**Comment by friday** [comment]
Sorry. I should have been more clear. The other issue relates to this, but I never actually tested `toggleControls()` with my commit. I had a look at it now, and it didn't fix things. The logic in toggleControls needs to be rewritten to be used externally. Many conditions and things don't add up.

**Comment by sampotts** [comment]
Yeah, it's pretty crazy. I think I've re-written it a few times but will take a look.

**Comment by friday** [comment]
This should works in the meanwhile:
```js
player.elements.container.classList.toggle('plyr--hide-controls');
```

**Comment by itpablo** [comment]
Hi @friday !

I'm trying your tip, but player.elements.**container** seems to be null.

Do you know what it can be?

Thanks!

**Comment by friday** [comment]
It does work (try it on plyr.io). If not, try make sure `player` is the plyr instance and perhaps try some alternative options.

`player.elements.container` is the plyr container element which you can get in other ways though.

Ex:
```js
document.querySelector('.plyr').classList.toggle('plyr--hide-controls');
```

**Comment by friday** [comment]
Fixed in v3.3.8
